Inventory.py

```diff
@@ -0,0 +1,40 @@
+import curses
+
+
+class Item(object):
+    def __init__(self, name, heal_value):
+        self.name = name
+        self.heal_value = heal_value
+
+
+class Equipment(object):
+    def __init__(self, name, strength):
+        self.name = name
+        self.strength = strength
+
+
+class Inventory(object):
+    def __init__(self):
+        self.items = {}
+        self.equipment = {}
+
+    # prints Inventory (placeholder)
+    def __str__(self):
```

Comment:
Hier sollte eigentlich eine Implementierung in curses stehen, nicht nur eine Ausgabe zu einem String